pkg/syntax/openapi/schema_writer_generated.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"flag"
+	"os"
+	"strings"
+
+	openapi "github.com/jenkins-x/jx/pkg/syntax/openapi/all"
+
+	"github.com/go-openapi/spec"
+
+	"github.com/pkg/errors"
+
+	"github.com/jenkins-x/jx/cmd/codegen/generator"
+)
+
+func main() {
```

Comment:
Where/How is this used? Should this maybe be part of the cmd/codegen tree?

Reply:
I have no idea - it just showed up. There's an equivalent in `pkg/client/openapi` as well.